packaging/linux/flatpak/scripts/remove-mouse.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+flatpak-spawn --host pkexec sh -c "gpasswd -d $USER input && rm /etc/udev/rules.d/85-sunshine.rules"
```

Comment:
Was this tested? Does `$USER` evaluate properly?

Reply:
Yes